src/file-events/cpp/jni_support.cpp

```diff
@@ -54,3 +88,57 @@ JniThreadAttacher::~JniThreadAttacher() {
         cerr << "Failed to detach JNI from current thread: " << ret << endl;
     }
 }
+
+JniConstants::JniConstants(JavaVM* jvm)
```

Comment:
Why are we adding a separate class for this one class? Shouldn't we just have one class where we have all these constants?

Reply:
I wanted to separate the JNI-only stuff from the native-platform related stuff.